content/client-lib-development-guide/features.textile

```diff
@@ -1242,6 +1258,31 @@ h4. TokenRequest
 * @(TE4)@ @ttl@ attribute represents time to live (expiry) of this token in milliseconds
 * @(TE6)@ @TokenRequest::fromJson@ static factory method that accepts either a @JsonObject@ or a string (which should be parsed as a JSON string), and returns a new @TokenRequest@. Statically typed languages (that expect the @authCallback@ to result in a typed @TokenRequest@ object, rather than, say, a hashmap) must implement this; others may at their discretion.
 
+h4. TokenRevocationRequest
+
+* @(TF1)@ @TokenRevocationRequest@ is a type containing the token revocation request details sent to the "REST revokeTokens endpoint":/rest-api/#revokeTokens
+* @(TF2)@ @TokenRevocationRequest#key@ attribute is a string representing key to revoke (appId.Key).
+* @(TF3)@ @TokenRevocationRequest#targets@ is an array of @TokenRevocationTarget@s to revoke. For request to be valid, it must contain at least one target. A test must exist to ensure there is at least one target in the request before it is sent.
+* @(TF4)@ @TokenRevocationRequest#issuedBefore@ is an optional timestamp (in milliseconds since the epoch) to revoke tokens issued before this time.
+* @(TF5)@ @TokenRevocationRequest#allowReauthMargin@ is an optional boolean that permits a token renewal cycle to take place without needing established connections to be dropped. Default for this value is false if not provided.
+
+h4. TokenRevocationTarget
+
+* @(TL1)@ @TokenRevocationTarget@ is a type representing a target for "REST revokeTokens endpoint":/rest-api/#revokeTokens.
+* @(TL2)@ @TokenRevocationTarget#key@ is a string containing the target key. Target key can be "clientId","revocationKey" or "channel". Where possible, clients should predefine those strings (enums for example) to prevent customers passing an invalid value.
+* @(TL3)@ @TokenRevocationTarget#value is a string representing the value of specified for target key. This value cannot be empty or null. 
```

Comment:
I don't understand this statement "is a string representing the value of specified for target key". Can you reword please?

Reply:
46bef24067f78d6812f1a98d93abe4126ccdf33f